bin/varnishd/mgt/mgt_cli.c

```diff
@@ -141,7 +141,20 @@ mcf_askchild(struct cli *cli, const char * const *av, void *priv)
 		    "Type 'help' for more info.");
 		return;
 	}
+
+	for (i = 0; i < ncmds; i++) {
+		if (strcmp(av[1], cmds[i]->request))
+			continue;
+		if (cmds[i]->flags & CLI_F_INTERNAL) {
```

Comment:
I think we shouldn't expose more than necessary. We are changing the CLI commands table in this very patch series, and I would personally prefer to keep the ability to change the table as we see fit without being concerned about API stability for end-users (especially in the context of back-ports to older branches).

For third-party utilities, I would refer to `help -j`, like `varnishadm` does since 38506b339af043d64bef12b7c27fbfc7df62e657.

Reply:
What is the point of exposing commands that can/should not be used anyway ? Internal commands are meant to be only issued by the MGT process, so in my opinion, nothing else should be aware of it.

Reply:
My point is that we should not make it harder than it already is to hunt down errors.